spec/ttml1.xml

```diff
@@ -4182,6 +4182,11 @@ as a style property only to those element types indicated in the following table
 </tbody>
 </table>
 <note role="clarification">
+<p>While permitted, the use of linear whitespace (LWSP) around the comma delimiters in the value of this 
+attribute is not recommended for maximal compatibilty with processor implementations.
```

Comment:
Is there a processor for which this is known to cause a problem?

Reply:
If we don't actually know of an incompatible processor, then please remove "... for maximal compatibility with processor implementations".